cjs/package.json

```diff
@@ -28,6 +28,6 @@
     "eslint": "^8.39.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-watch": "^8.0.0",
-    "nodemon": "^2.0.22"
+    "nodemon": "^3.0.1"
   }
```

Comment:
**Review comment 1:** Is there a specific reason why you upgraded nodemon for cjs and esm templates? @samuko-things

Reply:
> **Review comment 1:** Is there a specific reason why you upgraded nodemon for cjs and esm templates? @samuko-things

Oh! I believe it was during the npm install part. it asked me to update some part and some vulnerability things so I did. I'll downgrade it back. thanks.

Reply:
👍🏼